src/Identity/v3/Models/Token.php

```diff
@@ -116,6 +119,27 @@ public function create(array $data): Creatable
         }
 
         $response = $this->execute($this->api->postTokens(), $data);
-        return $this->populateFromResponse($response);
+        $token = $this->populateFromResponse($response);
+
+        // Cache response as an array to export if needed.
+        // Added key `id` which is auth token from HTTP header X-Subject-Token
+        $this->cachedToken = Utils::flattenJson(Utils::jsonDecode($response), $this->resourceKey);
+        $this->cachedToken['id'] = $token->id;
+
+        return $token;
+    }
+
+    /**
+     * Returns a serialized representation of an authentication token.
+     *
+     * Initialize OpenStack object using $params['cachedToken'] to reduce HTTP authentication call.
+     *
+     * This array is a modified version of POST response to `auth/tokens`. Do not manually modify this array.
```

Comment:
This array is a modified response of the POST request to `auth/tokens`

Reply:
The token is actually a `response`